faq.rst

```diff
@@ -399,62 +290,38 @@ How do I compare my local repository to a remote repository?
 To display the list of changes that are in your local repository, but not
 in the remote, use::
```

Comment:
Maybe clarify that this only lists commits that were copied from the remote the last time the local copy was updated (via “git pull” or “git fetch”)